apps/www/package.json

```diff
@@ -54,6 +54,7 @@
     "remark": "^13.0.0",
     "remark-gfm": "^3.0.1",
     "remark-html": "^15.0.1",
+    "shared-data": "*",
```

Comment:
if anyone is adding these, could put the '*' imports together in the list so its a bit obvious for anyone new that it's out the ordinary.

Reply:
I think npm install will resort them alphabetically